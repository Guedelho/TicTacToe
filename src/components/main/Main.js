import React from 'react';

import './Main.css';

const Main = ({children}) => (
  <main className="Main">
    {children}
  </main>
);

export default Main;
