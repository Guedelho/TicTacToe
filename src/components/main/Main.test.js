import React from 'react';
import { shallow } from 'enzyme';
import Main from './Main';

const children = <p>renders without crashing</p>;

it('renders without crashing', () => {
  expect(shallow(<Main children={children}/>)).toMatchSnapshot();
});