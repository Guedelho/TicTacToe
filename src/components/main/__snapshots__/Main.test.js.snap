// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Main>
    <p>
        renders without crashing
    </p>
</Main>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <p>
        renders without crashing
</p>,
      "className": "Main",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "renders without crashing",
      },
      "ref": null,
      "rendered": "renders without crashing",
      "type": "p",
    },
    "type": "main",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <p>
          renders without crashing
</p>,
        "className": "Main",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "renders without crashing",
        },
        "ref": null,
        "rendered": "renders without crashing",
        "type": "p",
      },
      "type": "main",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
